presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -243,4 +261,28 @@ private MaterializedResult getExpectedOrdersTableDescription(boolean dateSupport
                     .build();
         }
     }
+
+    @Test
+    public void testCreateSchema()
+            throws SQLException
+    {
+        skipTestUnless(canCreateSchema());
+        assertUpdate("CREATE SCHEMA test_create_schema");
+        assertQuery("SHOW SCHEMAS LIKE 'test_create_schema'", "VALUES 'test_create_schema'");
+        if (canDropSchema()) {
+            assertUpdate("DROP SCHEMA test_create_schema");
+        }
+        else {
+            cleanUpSchemas(ImmutableList.of("test_create_schema"));
+        }
+    }
+
+    @Test
+    public void testDropSchema()
+    {
+        skipTestUnless(canDropSchema());
```

Comment:
make sure that schema does not exist before creation, that it doesn't exists after deletion

Reply:
Also verify that you are unable to create schema when it exists, and to drop when it does not exist (in both tests).